app/web/src/newhotness/Explore.vue

```diff
@@ -1707,14 +1707,38 @@ watch(searchString, () => {
   });
 });
 
-// Clear the selection when the filter or search string changes
-// TODO leave the selection as long as it is still one of the filtered components?
-watch(filteredComponents, () => {
```

Comment:
i was like 99% sure we don't need this watcher at all... so this gives us a good reason to better understand that we can remove it